enlighten/Multispec.py

```diff
@@ -428,6 +447,15 @@ def update_widget(self):
         self.check_hide_others.setEnabled(multi)
         log.debug("update_widget end (%s)", self.device_id)
 
+    def readd(self, device_id):
```

Comment:
Is this method name a typo?  If it's meant to be other than "read", recommend a longer version.

Reply:
I'll change it to readd_spec_object. I shouldn't be using this anymore in the latest CEChanges. This was originally present because of what you described to me as David's desire for it to "quietly" reset. The idea here being that the high level spec object can persist while everything beneath it explodes and can be reconstructed with the reset device. The latest changes I kick off our spectrometer like you would see in the connection loop because it was overall easier and easier to work with. I think it would be valuable to keep this with a comment as it maybe something worth working on later so that we do get a quiet reset.